docs/ruleset_functions/get_zone_supply_return_exhaust_relief_terminal_fan_power_dict.md

```diff
@@ -0,0 +1,85 @@
+                - Apportion the relief kW to this zone based on the terminal unit primary flow for the zone and the total primary terminal unit flow across all zones that this HVAC system serves: `zone_total_relief_fan_power = zone_total_relief_fan_power + (hvac_sys_total_relief_fan_power * (zone_primary_air_flow/total_terminal_air_flow))`  
+
+            - Else, the HVAC system serves a single zone: `Else:`  
+                - Get dictionary of the hvac system's fan system's supply, return, exhaust and relief fan powers in kW: `fan_sys_kW = list(get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM(RMI,hvac.fan_system)["Fan_Power"])`  
+                - Get the hvac system total supply fan kW, need to get the first element in the list of fan powers:`hvac_sys_total_supply_fan_power = fan_sys_kW[0]` 
+                - Get the hvac system total return fan kW, need to get the second element in the list of fan powers:`hvac_sys_total_return_fan_power = fan_sys_kW[1]` 
+                - Get the hvac system total exhaust fan kW, need to get the third element in the list of fan powers:`hvac_sys_total_exhaust_fan_power = fan_sys_kW[2]` 
+                - Get the hvac system total relief fan kW, need to get the fourth element in the list of fan powers:`hvac_sys_total_relief_fan_power = fan_sys_kW[3]`     
+
+                - Add the supply kW to this zone: `zone_total_supply_fan_power = zone_total_supply_fan_power + hvac_sys_total_supply_fan_power`    
+                - Add the return kW to this zone: `zone_total_return_fan_power = zone_total_return_fan_power + hvac_sys_total_return_fan_power` 
+                - Add the exhaust kW to this zone: `zone_total_exhaust_fan_power = zone_total_exhaust_fan_power + hvac_sys_total_exhaust_fan_power`  
+                - Add the relief kW to this zone: `zone_total_relief_fan_power = zone_total_relief_fan_power + hvac_sys_total_relief_fan_power` 
+
+                - Get list of terminal units served by the hvac system: `terminal_list_hvac_sys_x = dict_of_zones_and_terminal_units_served_by_hvac_sys_x[hvac.id]["ZONE_LIST"]["Terminal_Unit_List"]`  
+                - For each terminal unit served by the HVAC system: `for terminal in terminal_list_hvac_sys_x:`  
+                    - Get the terminal unit fan power: `terminal_fan_power = get_fan_object_electric_power(RMI,terminal.fan)`  
+                    - Add to the total terminal kW associated with the zone: `zone_total_terminal_fan_power = zone_total_terminal_fan_power + terminal_fan_power`  
+
+        - Else, the hvac fan system equals Null meaning that the fan is defined at the terminal unit level like it is for a four pipe fan coil unit: `Else:`  
```

Comment:
Wouldn't we have fan systems both at the HVAC system level and terminal level for PVAV HVAC system?